File: repositorioacademico.client/src/app/models/sublinea-investigacion.ts
export interface SublineaInvestigacion {
  id: number;
  descripcion: string;
  estado?: string;
  lineaInvestigacionId: number;
  lineaInvestigacion?: string;
}

export interface CrearSublineaInvestigacionPayload {
  descripcion: string;
  lineaInvestigacionId: number;
  estado?: string;
}

File: RepositorioAcademico.Server/Domain/SublineaInvestigacion.cs
namespace RepositorioAcademico.Server.Domain
{
    public class SublineaInvestigacion
    {
        public int Id { get; set; }

        public int LineaInvestigacionId { get; set; }

        public LineaInvestigacion? LineaInvestigacion { get; set; }

        public string Descripcion { get; set; } = string.Empty;

        public string? Estado { get; set; }
    }
}
